style.css

```diff
@@ -205,8 +205,7 @@ p{
 }
 .section-testimonial-image{
   width: 50%;
-  background: url("https://picsum.photos/1000/900");
-  background-repeat: no-repeat;
+  background: url("https://picsum.photos/1000/900") no-repeat center center;
```

Comment:
Does this also need `background-size: cover;` ?

Reply:
Pushed the addition of background-size: cover;